app/api/reminders/route.ts

```diff
@@ -0,0 +1,124 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { prisma } from '@utils/prisma';
+
+// Configuration constants
+const UPLOAD_LINK = 'https://sistema-prod.vercel.app/calendar';
+const ADMIN_EMAIL = 'admin@sistema.ca';
```

Comment:
The admin email(s) should be all current users in the database with the admin role.